swarm/pss/README.md

```diff
@@ -0,0 +1,138 @@
+# Postal Service over Swarm
```

Comment:
This PR seems to include all of the pss work which has already been reviewed and merged into the `pss` branch in #123, was the intention to get it all reviewed again?